modules/33acrossAnalyticsAdapter.js

```diff
@@ -11,16 +11,19 @@ const { EVENTS } = CONSTANTS;
 const ANALYTICS_VERSION = '1.0.0';
 const PROVIDER_NAME = '33across';
 const GVLID = 58;
+/** Time to wait for all transactions in an auction to complete before sending the report */
 const DEFAULT_TRANSACTION_TIMEOUT = 10000;
+/** Time to wait after all GAM slots have registered before sending the report */
+const POST_GAM_TIMEOUT = 500;
```

Comment:
I was expecting to see a unit test that had a reference to this number, also just wanted the selection of the number (500ms) had some analysis before.. is it 500ms the minimum we need to wait for the registration?

Reply:
The 500ms makes sense.. the only thing that's missing is a unit test that covers the code wrapped by that setTimeout.

I was expecting to see a unit test that executes the GAM slot subscription code (basically executes the function pushed to cmd).. then the `slotRenderEnded` is triggered with an event that has the right ad unit path (a mocked getAdUnitPath) matching a transactionId and runs an assertion on the final result